### PR TITLE
order form: hide by default and show iff wallets are set.

### LINKED ISSUE #1114: Buy order misses after little while
When starting dexc into browser, it correctly shows the buy/sell form:

image

But after a little while, it hides itself (no action needed)

image

I think it might be happening because I do not have a btc wallet on it

### PR DESCRIPTION
**Pull request by amassarwi**
Closes #1114.

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]
Works great for me. I do not see the form until fee is confirmed and I have both wallets set up.

There are two typos in the original commit message.

**Comment by amassarwi** [comment]
@JoeGruffins thanks for the review, updated the commit message.

btw `iff` was intended - used in some code bases as `if and only if`. `be default` was indeed a typo.

**Review by vctt94 — APPROVED** [review]
Tested and works for me

File: client/webserver/site/src/js/markets.js [hunk 1]
```diff
@@ -514,23 +514,24 @@ export default class MarketsPage extends BasePage {
   /* setMarket sets the currently displayed market. */
   async setMarket (host, base, quote) {
     const dex = app.user.exchanges[host]
+    const { page } = this
```
Comment (buck54321) [inline client/webserver/site/src/js/markets.js:517]: `const page = this.page` is preferred for grabbing a single property.

File: client/webserver/site/src/js/markets.js [hunk 2]
```diff
@@ -1265,10 +1266,8 @@ export default class MarketsPage extends BasePage {
   setBalanceVisibility () {
     if (this.market.dex.connected) {
       Doc.show(this.page.balanceTable)
-      Doc.show(this.page.orderForm)
     } else {
       Doc.hide(this.page.balanceTable)
-      Doc.hide(this.page.orderForm)
     }
```
Comment (buck54321) [inline client/webserver/site/src/js/markets.js:1271]: Ah. This is the bug, huh.

**Review by buck54321 — APPROVED** [review]
One tiny nit. Nice fix.

**Pull request merged**